backend/task.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from sklearn.cluster import KMeans
+import pandas as pd
+import numpy as np
+import re
+import nltk
+import multiprocessing
+from gensim.models import Word2Vec
+from time import time  # To time our operations
+from collections import defaultdict  # For word frequency
+import logging  # Setting up the loggings to monitor gensim
+logging.basicConfig(format="%(levelname)s - %(asctime)s: %(message)s", datefmt= '%H:%M:%S', level=logging.INFO)
+
+import matplotlib.pyplot as plt
+
+data = pd.read_csv("floop_data_15k.csv")
+
+data.shape
+
+data.head()
+
+data.info()
+
+from string import punctuation
+
+punctuations = punctuation
+
+def solution(sentence):
+    for p in punctuations:
+        sentence = sentence.replace(p, '')
+    return sentence
+
+x = data["Field1"].apply(solution)
+pattern = "[^a-zA-Z0-9]"
+x_cleaned = [re.sub(pattern," ",text) for text in x]
+
+x_lowered = [text.lower() for text in x_cleaned]
+x_lowered
+
+x_lowered[0]
+
+nltk.download('punkt')
+
+x_tokenized = [nltk.word_tokenize(text) for text in x_lowered]
+
+nltk.download('wordnet')
+from nltk.stem import WordNetLemmatizer
+lemma = WordNetLemmatizer()
+
+x_lemmatized = [[lemma.lemmatize(word) for word in text] for text in x_tokenized]
+
+print(x_lemmatized[0])
+
+cores = multiprocessing.cpu_count()
+
+cores
+
+w2v_model = Word2Vec(min_count=20,
+                     window=2,
+                     size=300,
+                     sample=6e-5, 
+                     alpha=0.03, 
+                     min_alpha=0.0007, 
+                     negative=20,
+                     workers=cores-1)
```

Comment:
How long is this script supposed to run? Setting this worker value to `cores-1` means that you are using almost _all_ the host computers cpu capacity to do this work. If it's quick to execute then that might be ok, but anyone who runs this might not be able to run other programs while it's working

Reply:
It's a very light program so it required only One core. In my case my machine has only 2 cores. So that is why I use cores-1 but now, I have fixed it with 1, so from now onwards CPU will use one core